book/project_structure.md

```diff
@@ -101,7 +101,7 @@ A typical analytical project folder might look like:
 |   -- generate_plots.py
 ```
 
-``` {tab} R
+``` {code-tab} r R
```

Comment:
should this be r and R, or just R?

Reply:
"r" tells sphinx this is an R code tab. "R" is the title it displays.